### PR TITLE
Add CHANGELOG entry for username error enhancement

### PR DESCRIPTION
**Pull request by orenbm**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -13,6 +13,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - The project Golang version is updated from the end-of-life v1.12 to the latest
   version v1.15.
   [cyberark/conjur-authn-k8s-client#206](https://github.com/cyberark/conjur-authn-k8s-client/issues/206)
+- Improve the error message raised when the username doesn't include the `host/` prefix
```
Comment (abrahamko) [inline CHANGELOG.md:16]: changes should go at the top, no?

Comment (orenbm) [reply CHANGELOG.md:16]: i don't think that the order within every section is important.

Comment (orenbm) [reply CHANGELOG.md:16]: all the content is in the same release

Comment (abrahamko) [reply CHANGELOG.md:16]: ok, I thought it does.

**Review by abrahamko — APPROVED** [review]

**Pull request merged**